bigtable/quickstart/requirements.txt

```diff
@@ -1,2 +1,2 @@
-google-cloud-bigtable==1.0.0
-google-cloud-core==1.0.3
+google-cloud-bigtable==1.2.1
+google-cloud-core==1.1.0
```

Comment:
is the explicit dependency on core needed for something? Bigtable specifies   "google-cloud-core >= 1.0.3, < 2.0dev",